run-tests-with-build-optaplanner-quickstarts-parent/optaplanner-quickstarts-scm-checkout-test/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>run-tests-with-build-optaplanner-quickstarts-parent</artifactId>
+    <groupId>org.kie</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
+
+  <artifactId>optaplanner-quickstarts-scm-checkout-test</artifactId>
+
+  <properties>
+    <!-- Following are being passed to invoked projects as system properties using testPropertiesFile in invoker.
+         The entry in properties file is created only when the value for given property is passed using `-D` to the
+         overall maven build. -->
```

Comment:
So we need to pass all the properties bellow within the :

mvn clean verify \
-Dtest.type=scm-checkout \
-Drepository.url=https://github.com/kiegroup/optaplanner-quickstarts \
-Dsources.revision=8.11.x

Reply:
The overriding ones? Yes. Let's say I run run-tests-with-build command with `-Dnot-going-tobe-passed=XY -Dquarkus.platform.group-id=io.quarkus.platform` ... in this case just `quarkus.platform.group-id` should be passed, cause `not-going-tobe-passed` is not amongt the list of properties mentioned through the property.

Reply:
I specified this in readme.